### DISCUSSION TITLE
What Apache server version support php 5.6 - 8.3

### DISCUSSION THREAD

**Discussion by Witawat** [leokhoa/laragon#842] [Q&A]
hello

I have a program developed on php from 5.6 - 8.3.
I have a problem with Apache server every time I change the php version.

Sometimes there will be an error that cannot be used with the old php version.

I wonder if there is any way to fix this problem.

**Comment by jhzrmx** [comment]
Each Apache version only support certain kind of PHP version. If you want to change PHP versions, you would either:
- Change Apache version everytime you switch PHP version, or
- Use Nginx instead, with PHP-NTS.
